2nd-gen/packages/swc/.storybook/about-swc/when-to-use-swc.mdx

```diff
@@ -1,57 +1,74 @@
 import { Meta } from '@storybook/addon-docs/blocks';
+import headerImage from '../assets/images/banner_principles_S2_desktop_2x_1689964204945.png';
+import '@spectrum-web-components/accordion/sp-accordion.js';
+import '@spectrum-web-components/accordion/sp-accordion-item.js';
 
-<Meta title="When to use SWC?" />
+<Meta title="When to use SWC" />
+
+<img src={headerImage} alt="When to use SWC" class="docs-image" />
 
 # When to use SWC
 
 Spectrum Web Components (SWC) is a good choice when you want to build interfaces that match the Spectrum design system and can run in many different environments. This page explains when SWC is the right tool and when you may need extra support or a different approach.
 
-## Use SWC when:
-
-### You need Spectrum-aligned UI in a web environment
-
-SWC gives you ready-made components that follow Spectrum design and behavior. Use them when your project must look and feel like other Adobe experiences.
-
-### You want components that work without a specific framework
-
-SWC works in plain HTML, JavaScript, or TypeScript. It also works with frameworks like React through wrapper packages. This makes it a good option for teams using mixed technologies.
-
-### You want consistent behavior across products
-
-Because SWC components share a common architecture and API, they behave the same across all projects. This reduces the amount of one-off code your team needs to maintain.
+## SWC is a good choice when
 
-### You want built-in accessibility
+<sp-accordion allow-multiple size="m" class="docs-accordion">
```

Comment:
i override the border color with this class so match the hr under the h2. same issue with s1 variables not mapping